File: jscomp/test/boolean_test.js
'use strict';

Mt = require("./mt.js");
Test_bool_equal = require("./test_bool_equal.js");

Mt.from_suites("boolean", --[[ :: ]][
      --[[ tuple ]][
        "bool_equal",
        Test_bool_equal.assertions
      ],
      --[[ [] ]]0
    ]);

--[[  Not a pure module ]]

File: jscomp/test/boolean_test.lua
__console = {log = print};

Mt = require "..mt";
Test_bool_equal = require "..test_bool_equal";

Mt.from_suites("boolean", --[[ :: ]]{
      --[[ tuple ]]{
        "bool_equal",
        Test_bool_equal.assertions
      },
      --[[ [] ]]0
    });

exports = {};
return exports;
--[[  Not a pure module ]]
